### ISSUE TITLE
Feeds don't show the title or header at the top

### ISSUE THREAD

**Issue by steveharoz** [bluesky-social/social-app#2926]
**Describe the bug**
Feeds no longer show the feed title or header at the top. Even the buttons to unpin or unsave are gone.

**To Reproduce**

1. From the bsky.app (web) main screen, click one of the feeds from the column on the right.

**Expected behavior**

You should see info about the feed at the top. Here is a screenshot from going to the feed's URL:
![image](https://github.com/bluesky-social/social-app/assets/2257540/b6b6a080-5114-4d20-8922-076bbc243a56)

But if you click the list of pinned feeds, you only see "Bluesky":
![image](https://github.com/bluesky-social/social-app/assets/2257540/0b3be679-5320-49b9-9a61-865d2a372039)

**Details**

 - Platform: web - desktop Chrome
 - Platform version: Windows 11
 - App version: 1.68.0

**Comment by mary-ext** [comment]
This is intended, re: https://github.com/bluesky-social/social-app/pull/2775

**Comment by steveharoz** [comment]
Oh no! It seems like some use cases are now MUCH more challenging.

So now, there isn't a quick way to see the "About" info for a feed, which often has critical info like what hashtag or keywords to use and the rules of the feed.

There's also no quick way to send someone a link to a feed ("Hi fellow bird watcher. You might like the bird feed: [URL]").

**Comment by gaearon** [comment]
This is all true — note these are also all problems that already exist on mobile. So it might be good to figure out some holistic solutions to these. I agree we should make the header useful again though.
